cpp/src/arrow/flight/client.cc

```diff
@@ -58,7 +58,7 @@
 #include "arrow/flight/serialization_internal.h"
 #include "arrow/flight/types.h"
 
-namespace pb = arrow::flight::protocol;
+namespace afpb = arrow::flight::protocol;
```

Comment:
What if you put the "namespace pb" alias inside the `arrow` namespace below?
There is no reason to believe that top-level `afpb` won't conflict with another library.

Reply:
Yes, that worked. CI is green again.

Reply:
Yes, that worked. CI is green again.